help/bulk_data.md

```diff
@@ -63,3 +63,35 @@ dataset also includes extracted citation data.
 For all articles the [processed PDF and source files are available from
 Amazon S3](bulk_data_s3.md). We recommend this method for bulk access to
 the full-text of arXiv articles.
+
+<span id="harvest"></span>
+Custom Programmatic Harvesting 
+------------------------------
+
+As stated on our [robots](/help/robots) page, arXiv has limited server capacity 
+and our first priority is to support interactive use by human users. That said, 
+we are plainly aware that interested parties will want to make use of our corpus.
+
+### Play nice
+
+We ask that users intent on harvesting use the dedicated pages of `export.arxiv.org` 
+for these purposes. This will mitigate impact on readers who are using the site
+interactively. 
+
+There are hundreds (if not thousands) of users who want to make use of our data,
+and millions of distinct URL behind our site. If everyone were to crawl the site 
+at once without regard for a reasonable throttle, the site could be dragged down

```

Comment:
Should this be 'distinct URL**s**'

Is 'throttle' the prevalent term for client request rate controls? For me 'reasonable request rate' makes more sense than 'reasonable throttle'. Not a problem.